### ISSUE TITLE
futuresPositionRisk has a good chance of failing for some users

### ISSUE THREAD

**Issue by joelrich** [jaggedsoft/node-binance-api#421]
In the deployment yesterday, Binance has changed the behavior of the `positionRisk` API. Now, it will return something like this:

```
[
    {
        "entryPrice": "0.00000",
        "marginType": "isolated", 
        "isAutoAddMargin": "false",
        "isolatedMargin": "0.00000000", 
        "leverage": "10", 
        "liquidationPrice": "0", 
        "markPrice": "6679.50671178",   
        "maxNotionalValue": "20000000", 
        "positionAmt": "0.000", 
        "symbol": "BTCUSDT", 
        "unRealizedProfit": "0.00000000", 
        "positionSide": "BOTH",
    },
    {
        "entryPrice": "6563.66500", 
        "marginType": "isolated", 
        "isAutoAddMargin": "false",
        "isolatedMargin": "15517.54150468",
        "leverage": "10",
        "liquidationPrice": "5930.78",
        "markPrice": "6679.50671178",   
        "maxNotionalValue": "20000000", 
        "positionAmt": "20.000", 
        "symbol": "BTCUSDT", 
        "unRealizedProfit": "2316.83423560"
        "positionSide": "LONG", 
    },
```
The important part is the  `positionSide`. Suddenly, there can be (but now always be) multiple positions per symbol. The current code will just take the latest position - which may or may not be the right now.

I have had a similar issue with the user data stream (where LONG and SHORT is always included). I would suggest to add a new filter parameter `positionSide` with default value of BOTH. Then the old behavior is kept.

**Comment by jaggedsoft** [comment]
I see. Previously I mapped the return values to an array of symbols but this won't work anymore unless I return as an object like this:
```js
{
"LONG": { ... },
"SHORT: { ... },
"BOTH": { ... },
}
```
if that would be useful, I can do it that way. I will add a `raw` option to return the data exactly like how it comes from Binance as well.

**Comment by joelrich** [comment]
I think what is most important is that the old behavior is preserved. For this, I would just ignore everything else than BOTH (and `undefined` in case they roll back this change). `raw` sounds like a good addition.

**Comment by jaggedsoft** [comment]
Thank you for all of your help @joelrich it is very much appreciated.
Fixed in https://github.com/jaggedsoft/node-binance-api/commit/f408b3850075f3f97258a8f8bfff452a55e0d4d2
available in v0.11.1

 raw

**Comment by jaggedsoft** [comment]
I just realized after publishing this that your suggestion of a filter value was much better. I can fix this next version if necessary.
